src/App.vue

```diff
@@ -1,12 +1,37 @@
 <template>
     <Sidebar />
-    <LiveCanvas />
+    <template v-if="store.state.board">
+        <LiveBoard v-if="store.state.board.started"/>
+        <BoardLockedPlaceholder v-else />
+    </template>
+    <BoardLoadingPlaceholder v-else />
     <ErrorOverlay />
 </template>
 
 <script setup lang="ts">
-import Sidebar from "./components/Sidebar.vue";
-import LiveCanvas from "./components/LiveCanvas.vue";
+import Sidebar from "./components/sidebar/Sidebar.vue";
+import LiveBoard from "./components/LiveBoard.vue";
 import ErrorOverlay from "./components/error/ErrorOverlay.vue";
+import { onMounted } from "vue";
 
+import AzPlaceAPI from "@/api.js";
```

Comment:
1. why "api.js", the file is named "api.ts"
2. afaik you can just wirte "@api" without the file extension